pkg/sync/rpc/server.go

```diff
@@ -344,8 +344,8 @@ func (s *SyncAgentServer) FilesSync(ctx context.Context, req *ptypes.FilesSyncRe
 			ops = s.RebuildStatus
 		}
 
-		port, err := s.launchReceiver("FilesSync", info.ToFileName, ops)
-		if err != nil {
+		var port int
+		if port, err = s.launchReceiver("FilesSync", info.ToFileName, ops); err != nil {
```

Comment:
Don't need to change this part now.

`err` will be the one updated by the `return` statement.

Should add `:` to the `err := fromClient.SendFile()` below.